onCallRotation.md

```diff
@@ -4,8 +4,8 @@ This document is used to schedule and track staff coverage for unforeseeable ope
 
 Our Service Level Agreement (SLA) requires continuous support staffing during customer business hours: 9:00 am to 5:00 pm, Monday through Friday. 
 
-- When your shift starts, check the `help@dewv.net` mail inbox, and monitor it until your shift ends. Check the recent entries in the Duty Log below for any relevant information from the shift(s) preceding yours. Add an entry *to the top* of the Duty Log to generate a GitHub contribution, using `Report for duty` as the commit message.
-- Respond to mail messages for new customer requests, using appropriate message templates ("signatures"). Templates include instructions for creating an Issue in this repo. At the same time, you should create a mailbox folder named `IssueX`, where `X` is the new Issue number.
+- When your shift starts, check the `help@dewv.net` mail inbox by monitoring your personal email through the outlook group, and monitor it until your shift ends. Check the recent entries in the Duty Log below for any relevant information from the shift(s) preceding yours. Add an entry *to the top* of the Duty Log to generate a GitHub contribution, using `Report for duty` as the commit message.
```

Comment:
Good changes. It could use some more, because there is still text that talks about filing the mail, the Reminders feature, etc.-- stuff that applies to the ionos web mail but not to Office 365 group.